### PR TITLE
Design changes to Firm Edit page

### PR DESCRIPTION
**Pull request by neoeno**
## Before

image

## After

image

@munckymagik

### CODE REVIEW

**Comment by asmega** [comment]
:+1:

**Comment by JiggyPete** [comment]
:+1:

File: app/views/self_service/firms/edit.html.erb [hunk 1]
```diff
@@ -3,14 +3,19 @@
...
 
 <div class="l-constrained">
-  <h1 class="t-firm-name">
-    <%= @firm.registered_name %>
-  </h1>
+  <div class="l-heading-with-inline-links">
+    <h1 class="t-firm-name">
+      <%= @firm.registered_name %>
+    </h1>
+    <% if @firm.persisted? %>
```
Comment (neoeno) [inline app/views/self_service/firms/edit.html.erb]: Things in an `edit` partial are always going to be persisted.

So this also needs to go into the create template, now.

File: app/views/self_service/trading_names/edit.html.erb [hunk 2]
```diff
@@ -1,12 +1,16 @@
 <div class="l-constrained">
-  <h1 class="t-firm-name">
-    <%= @firm.registered_name %>
-  </h1>
+  <div class="l-heading-with-inline-links">
+    <h1 class="t-firm-name">
+      <%= @firm.registered_name %>
+    </h1>
+    <% if @firm.persisted? %>
```
Comment (neoeno) [inline app/views/self_service/trading_names/edit.html.erb]: Same

**Pull request merged**